contract_sale/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Contract from Sale',
+    'version': '12.0.1.0.0',
+    'category': 'Sales',
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/contract',
+    'depends': [
+        'sale',
```

Comment:
IMHO I think we should depends on sale_management because if not you have in any case to install sale_management always as without this module you don't have any menu sales.

Reply:
I don't think so. These are technical dependencies that prevents the module to work, but not real ones. Maybe a comment can be dropped in the installation section for testing the module.